### ISSUE TITLE
Zombie Dogs Not Running

### ISSUE THREAD

**Issue by MooTorial** [Zet0rz/nzombies#632]
As the title states, zombie dogs don't seem to be running for me. I've downloaded a few configs off the workshop, as well as my own.

**Comment by Charcoal87** [comment]
Do you have a working navmesh on the map/config? If so, then the spawns must be placed in an area where they get stuck as soon as they spawn

**Comment by MooTorial** [comment]
Yeah, working everything actually. The dogs walk everywhere instead of running. Just looks kinda dumb and plays really weird as well.

**Comment by Zet0rz** [comment]
Dogs walk until they get within a certain range of their target. Each dog targets a specific player.

**Comment by MooTorial** [comment]
I've tried multiple maps in both solo and multiplayer, still no running dogs.

**Comment by DjNathann** [comment]
@Zet0rz Yea dogs aren't running anymore they only walk to the target.

**Comment by Zet0rz** [comment]
It's weird, since the AI for the dogs haven't updated in a long time and they worked last iteration. Maybe a gmod update broke them? Can you confirm this happened suddenly?

**Comment by minin43** [comment]
I can confirm they've recently started acting unusual.

**Comment by globalcringe** [comment]
Can confirm as well, really strange bug.

**Comment by DjNathann** [comment]
Its very rare for dogs to run at their target but you'll mostly see dogs just walking to their target or you.

**Comment by CruntDUCKY** [comment]
![ze](https://user-images.githubusercontent.com/39637178/40570694-eb7396e0-6052-11e8-80fd-e59c9f008f48.gif)

It looks like they shat on the carpet and are feeling guilty about it.
2018.03.15pls fix thx
Also melee don't do shiet to dogs

**Comment by DjNathann** [comment]
Dogs have bad hitboxes hopefully the next major update will fix the Walking dogs bug and the shit hitbox.

**Comment by CruntDUCKY** [comment]
![wtf](https://user-images.githubusercontent.com/39637178/40869148-546d5b76-65db-11e8-85a3-960f13f6f03b.gif)

Now it's just pulling my goddamn leg. 

They **are** taking a shit.

The combination of widows wine and the jet gun (or any other weapons) outputs this retard hound.
Widows wine is probably the cause of these issues with hounds not running and other stuff.
2018.03.28pls fix thx
